pages/common/fossa.md

```diff
@@ -0,0 +1,24 @@
+# fossa
+
+> Fossa CLI - Integrate realtime license audits, vulnerability scans and reports about dependencies licenses.
```

Comment:
Maybe "Fossa CLI" --> "CLI for the Fossa service"? Otherwise it sounds like we're just saying this is a CLI named Fossa, which is redundant.

Also, I'd probably replace "integrate" with "generate", "produce" or something alike.

Reply:
Applied!